frontend/src/app/modules/common/hide-section/add-section-dropdown/add-section-dropdown.component.ts

```diff
@@ -27,18 +27,18 @@
 // ++
 
 import {I18nService} from "core-app/modules/common/i18n/i18n.service";
-import {Component, ElementRef, OnDestroy, OnInit, ViewChild} from "@angular/core";
-import {untilComponentDestroyed} from "ng2-rx-componentdestroyed";
+import {Component, ElementRef, OnInit, ViewChild} from "@angular/core";
 import {HideSectionDefinition, HideSectionService} from "core-app/modules/common/hide-section/hide-section.service";
 import {AngularTrackingHelpers} from "core-components/angular/tracking-functions";
+import {UntilDestroyedMixin} from "core-app/helpers/angular/until-destroyed.mixin";
 
 export const addSectionDropdownSelector = 'add-section-dropdown';
 
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><del><span data-href="https://github.com/opf/openproject/blob/85e9c35a9c9361e0038637188a645cba25065d78/frontend/src/app/modules/common/hide-section/add-section-dropdown/add-section-dropdown.component.ts#L36"></span>Please use <code>ChangeDetectionStrategy.OnPush</code> for this component</del></td>
    </tr>
  </tbody>
</table>